public/usage-examples/geometry/circle_y/circle_y-1-simple-oop.cs

```diff
@@ -0,0 +1,35 @@
+using SplashKitSDK;
+
+namespace CircleY
+{
+    public class Program
+    {
+        public static void Main()
+        {
+        SplashKit.OpenWindow("Circle Y", 800, 600);
+        SplashKit.ClearScreen();
+
+        Random random = new Random();
+        // Set position for the circle
+        double x_position = 400;
+        //Give random  y_position value bewteen 200 - 400
+        double y_position = random.Next(200) + 200;
+
+        // Create A circle name "A"
```

Comment:
Here in both c sharp files this comment is 
`// Create A circle name "A"` but it is different in the python and cpp files